index.html

```diff
@@ -1,160 +1,279 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-    <meta charset="UTF-8">
-    <meta http-equiv="X-UA-Compatible" content="IE=edge">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>My Portfolio</title>
-    <link rel="stylesheet" href="styles.css">
-</head>
-<body>
-    
-    <nav class="menu">
+    <link rel="stylesheet" href="styles.css" />
+    <link rel="stylesheet" href="desktop.css" media="screen and (min-width: 800px)"
+    />
+  </head>
+  <body>
+    <header>
+      <nav class="menu">
         <a href="#" class="logo">MIRNA</a>
-        <img src="./image/Enabled.png" alt="hamburger icon" class="hamburger_icon">
-    </nav>
-    <section class="header">
-        <h1 class="header_title">I'm Mirna, welcome to my portfolio!</h1>
-        <p>I'm a software developer! I can help you build a product, feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don't hestiate to contact me.</p>
+        <img class="hamburger" src="./image/Enabled.png" alt="hamburger">
+        <ul class="nav-menu">
+          <li class="links">Portfolio</li>
+          <li class="links">About</li>
+          <li class="links">Contact</li>
+        </ul>
+        </div>
+        
+      
+    </header>
+    <main>
+      <section class="header">
+        <h1 class="header_title">I'm Mirna, <br class="space">  welcome to my portfolio!</h1>
+        <p class="header-presentation">
+          I'm a software developer! I can help you build a product, feature or
+          website Look through some of my work and experience! If you like what
+          you see and have a project you need coded, don't hestiate to contact me.
+        </p>
```

Comment:
- [ OPTIONAL ] As I can see here the margin/padding is not like the figma design but it should be equal from the top and from the bottom so kindly work on it a little bit to have the same margin/padding to match the figma as much as possible 💪🏻✨🔥💯🚀

- I will show you tow pictures so you can see the difference 🚀🚀

- Your design
![image](https://user-images.githubusercontent.com/58553711/193432715-cfb18b0a-598f-411e-8153-1130be8d9e68.png)
 
- Figma design
![image](https://user-images.githubusercontent.com/58553711/193432721-9d0ed1dd-ceb8-450b-a3f5-470fd3022680.png)